README.md

```diff
@@ -1,3 +1,11 @@
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
```

Comment:
The link attached is wrong here (takes us to anitab-org website chat)please use this link https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs